exec/plugin-commands-script-runners/src/regexpCommand.ts

```diff
@@ -0,0 +1,15 @@
+export function tryBuildRegExpFromCommand (
+  command: string
+): RegExp | null {
+  if (command.length < 3) {
+    return null
+  }
+  if (command[0] !== '/' || command.lastIndexOf('/') < 1) {
+    return null
+  }
+  try {
+    return new RegExp(command.slice(0, command.lastIndexOf('/')).slice(1))
+  } catch {
+    return null
+  }
+}
```

Comment:
Keep blank line endings.
```suggestion
}

```

Reply:
[421a693](https://github.com/pnpm/pnpm/pull/5871/commits/421a693ae25ff2900f74ea58662f3f25ecbdb771) Thanks!